src/helpers/filters.ts

```diff
@@ -64,6 +64,26 @@ const fileterByToAmount = (transactions: Transaction[], toAmount: number) => {
   );
 };
 
+export const filterBySearch = (transactions: Transaction[], search: string) => {
+  return transactions.filter(transaction => {
+    const inputs: string[] = [];
+    const outputs: string[] = [];
+    transaction.inputs.filter(input => {
+      inputs.push(...input.addresses);
+    });
+    transaction.outputs.filter(output => {
```

Comment:
`filter` shouldn't be used for that I think

have you considered using Array.reduce and without `Array.push` 
or `Array.forEach` with `Array.push`?

Reply:
although Array.reduce would be the best